exporters/failure/collector_github.py

```diff
@@ -179,7 +179,7 @@ def search_issues(self) -> list[TrackerIssue]:
         return critical_issues
 
     def get_app_name(self, issue, label: Optional[dict[str, Any]]):
-        if label:
+        if label and "=" in label["name"]:
             return label["name"].split("=")[1]
         # default to repo name if app_label is not set
         else:
```

Comment:
question (for issue or other PR): in Jira if there is no label (or wrong label) is goes with `unknown`. Should we do the same for GitHub? (instead of the github repo name)

Reply:
In github the issues are tied to the repo, that is not the case w/ jira.  If we had a better fallback in jira I would like that better than "unknown".  IMHO the repo name > "unknown"